components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/API.java

```diff
@@ -766,8 +768,47 @@ public void removeAllPolicies() {
         availableSubscriptionLevelPolicies.clear();
     }
 
-    public void removeAvailableTiers(Set<Tier> availableTiers) {
-        this.availableTiers.removeAll(availableTiers);
+    public Set<OrganizationTiers> getAvailableTiersForOrganizations() {
+        return availableTiersForOrganizations;
+    }
+
+    public void setAvailableTiersForOrganizations(Set<OrganizationTiers> availableTiersForOrganizations) {
+        this.availableTiersForOrganizations = availableTiersForOrganizations;
+    }
+
+    public void removeAllTiersForOrganizations() {
+        availableTiersForOrganizations.clear();
+    }
+
+    public void setAvailableTiersForOrganizationsFromString(String tiersString) {
+
```

Comment:
lets move logic out from the model class

Reply:
Resolved by https://github.com/wso2/carbon-apimgt/pull/12812/commits/d3c4cb68420e47b76a5d77f732f4ddb17fd090f3